gcp/README.md

```diff
@@ -151,41 +151,18 @@ Workflow: We will create a notebook instance, and run a shell script that will i
   
```

Comment:
I think the 'Pull' section just above this 'Build' area does this. The CLI instructions work the same either way, because they just specify the gcr.io image in the job description.

Reply:
This would be similar to the `Install RAPIDS on a pre-made Notebook` section, instead of installing RAPIDS in an env would be able to use the DLVM+RAPIDS env, just pinged GCP and they're working on updating the UI to choose from dropdown